.github/workflows/main.yaml

```diff
@@ -0,0 +1,20 @@
+name: Test gha-secret-manager
+on:
+  - push
+
+jobs:
+  secret_manager:
+    runs-on: ubuntu-latest
+    name: Test gha-secret-manager
+    steps:
+      - name: Test gha-secret-manager
+        id: secrets
+        uses: dmsi-io/gha-secret-manager@feature/KNOX-242
+        with:
+          GCP_SA_KEY: ${{ secrets.GCP_SECRET_MANAGER_KEY }}
```

Comment:
This is gonna break again due to being a public repo, right?

Reply:
I added local secrets to this repo